### ISSUE TITLE
If .stylelintrc is not in project root folder, nothing gets formatted.

### ISSUE THREAD

**Issue by Bubbi** [1000ch/atom-cssfmt#12]
I think this is an atom-stylefmt issue, but please correct me if i'm wrong.

If I open a project folder in Atom and uses atom-stylefmt to format the code, then it won't format anything unless the .stylelintrc file is in the project root folder.

If I manually type in commandline, `stylefmt [file 1] [file 2]`, then it will format the file according to the specified rules just fine.

I use linter-stylelint to mark errors and show error descriptions inline, and this works just fine if the .stylelintrc file is in the same folder or any parent folder of the css/scss files.

Is there a way to make atom-stylefmt work the same way?

**Comment by gucong3000** [comment]
I am try to fix this issue and #13 

First  I need to convince @morishitter to merge with my [PR](https://github.com/morishitter/stylefmt/pull/221)

Then `atom-stylefmt` needs to merge #16

**Comment by gucong3000** [comment]
@Bubbi ,Can you help me do some test?

replace `~/.atom/packages/stylefmt/index.js` with `https://github.com/gucong3000/atom-stylefmt/blob/patch-1/index.js`

and run

``` bash
cd ~/.atom/packages/stylefmt/
npm i gucong3000/stylefmt#sync_stylelint_api --save
```

Then you can try format your code
